shm/inc/player.hpp

```diff
@@ -17,6 +18,8 @@ class Player {
     Cargo getCargo(size_t index) const { return ship_->getCargo(index); }
 
 private:
+    size_t calculateavailableSpace();
```

Comment:
```suggestion
    size_t getAvailableSpace();
```
or `calculateAvailableSpace()`